.gitignore

```diff
@@ -46,3 +46,6 @@ Network Trash Folder
 Temporary Items
 .apdisk
 notes.txt
+
+# Test
+test.js
```

Comment:
I'd prefer if you tracked this locally rather than add it to the repo's .gitignore. (e.g. in case anyone wants to add tests in the future.)